src/dpo/test/ibex.ok

```diff
@@ -59,7 +64,9 @@ Importing netlist into detailed improver.
 Detailed Improvement Results
 ------------------------------------------
 Original HPWL           652460.9 u
-Final HPWL              639841.3 u
-Delta HPWL                  -1.9 %
+Final HPWL              636304.9 u
+Delta HPWL                  -2.5 %
 
-No differences found.
+Differences found at line 5426.
+    - _27724_ AND3_X1 + PLACED ( 975340 826000 ) N ;
+    - _27724_ AND3_X1 + PLACED ( 970020 834400 ) FS ;
```

Comment:
You need to update the defok file so that no differences are report.  Similarly with the other tests.

Reply:
I am getting different def file each time, can it be because of the change of parameters so the def came out a bit different each run?

In the old set of command the run time was ~10sec and in the new it is ~40sec so it is big diff.
I can also reduce the -f flag to 50 it will still give better results but reduce the run time to ~15-20 sec instead.

I didn't try routing it later, I will try it on some design and update.